website/docs/foundations/typography/partials/code/how-to-use.md

```diff
@@ -1,64 +1,58 @@
-The suggested way to apply the typographic definitions to a UI element is using the **predefined CSS helper classes** provided.
+## How to use these styles
 
-#### Design tokens
+While we do offer design tokens for typography, we recommend using the predefined CSS helper classes.
 
-There are numerous [typographic design tokens](./tokens) in our system.
+### CSS helper classes
 
-Since they are "atomic" definitions that associate a particular typographic property (e.g. `font-size`, or `line-height`, or `letter-spacing`) to a single value, they are not exactly "typographic" styles.
+In a single declaration, these classes contain everything to apply a standard style to an element: `font-family`, `font-size`, `line-height`, and a reset for `margin` and `padding` to match the intended design language.
 
-For this reason we **strongly advise against** using them directly in your CSS, and prefer the CSS helpers provided by the system, to avoid "mix & match" of typographic styles.
+```markup
+<p class="hds-typography-display-300">The fox jumped over the lazy dog.</p>
+```
 
-#### CSS helper classes
+#### Font styles
 
-There are different CSS helper classes that can be used for different purposes.
+<Doc::FontHelpersList @items={{this.cssHelpers.styles}} />
 
-If you want to change **only** the _font-family_ you can use one the "font-family" helpers:
+#### Font family
+
+While we don’t recommend using custom styles often, `font-family` helpers are available to change the font family of an element.
 
 ```markup
-<p class="hds-font-family-sans-text">Lorem ipsum dolor.</p>
+<p class="hds-font-family-sans-text">The fox jumped over the lazy dog.</p>
 ```
 
-These helpers are meant to use in very special cases, when the designers used custom typographic styles.
-
-_Notice: the "font-family-sans-display" helper is intented for headings and titles, while the "font-family-sans-text" helper is intended for body copy text. The "font-family-mono-code" helper is intended for monospaced text._
+| `font-family` helpers         | Use for                  |
+| ------------------------------| ------------------------ |
+| `font-family-sans-display`    | Headings and titles      |
+| `font-family-sans-text`       | Body copy                |
+| `font-family-mono-code`       | Monospaced text          |
 
-In most of the cases, you will use the _typography_ CSS helpers:
 
-```markup
-<p class="hds-typography-display-300">Lorem ipsum dolor.</p>
-```
+<Doc::FontHelpersList @items={{this.cssHelpers.families}} />
 
-These classes will contain, in a single declaration, everything that you need to apply a "standard" style to an element: _font-family_, _font-size_, _line-height_, plus a reset for _margin_ and _padding_ to `0px` (to match how they behave in Figma).
+#### Font weight
 
-If you want to change the _font-weight_ of an element you can use one the "font-weight" helpers:
+Use the `font-weight` helpers to change the weight of an element.
```

Comment:
Would we want to clarify that it changes the weight of "text" vs. "an element"?

(Maybe say: "changes the weight of text in an element"?)